EdgeOfEmpireBot/EdgeOfEmpireBot/MessageHandlers/Router/MessageRouter.cs

```diff
@@ -66,6 +67,7 @@ private async Task RouteCommand(string userInput)
     /// </returns>
     private async Task<string> GetProcess(string command)
     {
+        try{
```

Comment:
@Elliott127  should I remove these additional try catch blocks? 
These additional blocks were really only here to understand the error output when I thought I was deploying to a screen and not a Demon itself. So probably fine to remove? though I do like it outputting errors for the moment, thoughts?

Reply:
Yeah I think the TryGet already handles the error so it should be fine to remove